drivers/stepper/gpio_stepper_controller.c

```diff
@@ -22,11 +22,11 @@ static const uint8_t
 		{1, 1, 0, 0}, {0, 1, 0, 0}, {0, 1, 1, 0}, {0, 0, 1, 0},
 		{0, 0, 1, 1}, {0, 0, 0, 1}, {1, 0, 0, 1}, {1, 0, 0, 0}};
 
-struct gpio_stepper_motor_controller_config {
+struct gpio_stepper_config {
```

Comment:
can you move the struct rename in their own commit? then the per-commit diff makes more sense